governance/votes/Executive Vote - November 12, 2021.md

```diff
@@ -0,0 +1,84 @@
+
+$OfficeHours? This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### GUNIV3DAIUSDC-A Parameter Adjustments
+
+As per [this poll](https://vote.makerdao.com/polling/QmemHGSM?network=mainnet), the following GUNIV3DAIUSDC-A parameter adjustments will occur, if this executive passes:
+
+* Increase the [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam) from 50 million DAI to 500 million DAI
+* Decrease the [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio) from 105% to 102%
+* Decrease the [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee) from 1% to 0.5%
+
+For more information on this proposal, please see this [discussion thread](https://forum.makerdao.com/t/request-to-raise-the-guniv3daiusdc1-a-dc-to-500m/11394).
+
+### WSTETH-A Debt Ceiling Increase
+
+As per [this poll](https://vote.makerdao.com/polling/QmeQUKFm?network=mainnet), the following change will occur, if this executive passes:
+
+* Increase the [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam) from 5 million DAI to 50 million DAI
+
+For more information on this proposal, please see this [discussion thread](https://forum.makerdao.com/t/request-to-raise-staked-eth-dc-to-50m/11402).
+
+### DIRECT-AAVEV2-DAI Parameter Adjustments
+
+As per [this poll](https://vote.makerdao.com/polling/QmNbTzG1?network=mainnet#poll-detail), the following DIRECT-AAVEV2-DAI parameter adjustments will occur, if this executive passes:
+
+* Increase the [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam) from 10 million DAI to 50 million DAI
+* Increase the [Target Available Debt (gap)](https://makerdao.world/en/learn/governance/module-dciam) from 10 million DAI to 25 million DAI
+* Decrease the Target Borrow Rate from 4% to 3.9%
+
+For more information on this proposal, please see this [discussion thread](https://forum.makerdao.com/t/d3m-dc-increase-target-borrow-rate-decrease-proposal/11421).
+
+### Return Unused DAI to the Surplus Buffer
+
+As per this [forum thread](https://forum.makerdao.com/t/govalpha-budget-return/11341), 218,059.81 DAI will be returned to the Surplus Buffer from PauseProxy, if this executive passes. 
+
+This amount comprises budgeted funds that were unused by the GovAlpha Core Unit.
```

Comment:
Maybe link this to the GovAlpha mandate MIP on the mips portal.

Reply:
Could you elaborate which bit you want linked to the mandate?

Reply:
I usually do [GovAlpha Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP3)